### PR TITLE
docs(README): add compatibility section

### PR DESCRIPTION
**Pull request by GabrielDelepine**
and clarify the case of webpack 4

### CODE REVIEW

**Comment by GabrielDelepine** [comment]
I have signed the CLA

**Comment by garygreen** [comment]
It might be better to link to the more 'official' Webpack 4 compatibility discussion https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/749

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@
   <p>Extract text from a bundle, or bundles, into a separate file.</p>
 </div>
 
+<h2 align="center">Compatibility</h2>
+
+> :warning: Webpack 4 is not supported. Please check [this comment](https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/731#issuecomment-374548865) for futher information
```
Comment (bebraw) [inline README.md]: futher -> further

**Comment by michael-ciniawsky** [comment]
Superseded by https://github.com/webpack-contrib/extract-text-webpack-plugin/pull/777

**Pull request closed without merging**